### ISSUE TITLE
[Features]Instructions for new users

### ISSUE THREAD

**Issue by hanshou101** [racket/games#14]
I am a new Racket user and I want to try these games. Where should I start?

I use Google to search, but a lot of "Racket Games Guide" related content is directed to Racket Sport. This makes my search inefficient.

On the ReadMe.md of this GitHub repository, I have not found anything that can help.

In the following link, there is no related content: https://docs.racket-lang.org/games/index.html

May I ask if I want to try [Games: Fun Examples], where should I start? Or what kind of reference materials should be used?

**Comment by samth** [comment]
I'm not sure what you are looking for. The page you linked has documentation about each of the games. Are you having trouble starting the games? Or playing them?

**Comment by soegaard** [comment]
If you are using macOS or Windows look for the executable "PLT Games" in the same folder as DrRacket is placed (or simply search for "PLT Games" in Spotlight or what-the-search-for-file-feature is called in Windows).

**Comment by hanshou101** [comment]
> If you are using macOS or Windows look for the executable "PLT Games" in the same folder as DrRacket is placed (or simply search for "PLT Games" in Spotlight or what-the-search-for-file-feature is called in Windows).

-------------------------------

Thank you very much, sir! By your way, I have successfully run the game!
I will try to understand each line of code against the actual game running!

Thank you very much. English is not my first language, so I am reading the official Racket documentation, but the progress is relatively slow; I spend half an hour every day reading the documentation, but I have not read the content of "PLT Games".

So I failed to clearly understand the problem I faced before I didn't get your help.

Thank you for answering my question! Sir!
